### PR TITLE
[se-fornmin] Finalize processing code

### PR DESCRIPTION
**Pull request by Vesihiisi**
Make sure that the preview page is ready for release:

https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/se-fornmin_(sv)/preview

Add reporting of unmatched values.

Task: https://phabricator.wikimedia.org/T160617

### CODE REVIEW

File: importer/SeFornminSv.py [hunk 1]
```diff
@@ -138,17 +139,6 @@ def set_location(self):
             self.add_statement("location", self.get_socken(
```
Comment (lokal-profil) [inline importer/SeFornminSv.py]: Should there be an `add_to_report` here in case matching fails?

Comment (Vesihiisi) [inline importer/SeFornminSv.py]: Added it.

Comment (Vesihiisi) [inline importer/SeFornminSv.py]: Re: 1 -- I'm not sure which part exactly you mean by 'parser'?

Comment (lokal-profil) [inline importer/SeFornminSv.py]: I mean in `utils.get_wikilinks()` which in turn calls `mediawikiparserfromhell`. So if you send it a string without `[[` it should just return an empty list?

Comment (Vesihiisi) [inline importer/SeFornminSv.py]: Ah, right. I've cleaned it up.

**Pull request merged**